src/flex/FlexView.js

```diff
@@ -140,7 +142,11 @@ export default React.createClass({
   render() {
     const className = this.getClasses();
     const style = this.getStyle();
-    return <div className={className} style={style}>{this.props.children}</div>;
+    return (
+      <div className={className} style={style} {...this.props.events}>
```

Comment:
maybe better to omit `className` and `style` from `props.events`? Duplicate props can cause unexpected behavior